spec/add_gist_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'add_gist'
+
+describe AddGist do
+
+  describe '.read_files' do
```

Comment:
We should test the public method that calls this private method, since the private methods are not designed to be accessible outside of the class, and therefore reflect how the class would be used.